SimPEG/EM/TDEM/ProblemTDEM.py

```diff
@@ -650,10 +647,170 @@ class Problem3D_e(BaseTDEMProblem):
     _formulation = 'EB'
     fieldsPair = Fields3D_e  #: A Fields3D_e
     surveyPair = SurveyTDEM
+    Adcinv = None
 
     def __init__(self, mesh, **kwargs):
         BaseTDEMProblem.__init__(self, mesh, **kwargs)
 
+    def Jtvec(self, m, v, f=None):
```

Comment:
with this, I would prefer to generalize the `Jvec` in the base class to handle this case rather than implementing a separate `Jvec`. Would you like to take a crack at that @sgkang? or I would be happy to - up to you